test/acceptance/create_webdriver.robot

```diff
@@ -15,38 +15,54 @@ Create Webdriver Creates Functioning WebDriver
 
 Create Webdriver With Bad Driver Name
     [Documentation]    Invalid browser name
-    Run Keyword And Expect Error    'Fireox' is not a valid WebDriver name
-    ...    Create Webdriver    Fireox
+    Run Keyword And Expect Error    'Fireox' is not a valid WebDriver name    Create Webdriver
+    ...    Fireox
```

Comment:
Cleanup to code is fine bu unrelated changes should be submitted as separate PRs. In this case the code doesn't actually get any better, on the contrary, so this change should be reverted.